OpenTabletDriver.Configurations/Configurations/Huion/H951P.json

```
@@ -0,0 +1,39 @@
+{
+  "Name": "Huion H951P",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 221.0,
+      "Height": 141.0,
```

Based off the MaxX and MaxY values, these settings are likely more correct (matching 200 LPMM exactly):
```suggestion
      "Width": 220.985,
      "Height": 137.995,
```

---

MaxX and MaxY are slightly off, I added the right values as another suggestion